.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# DO NOT EDIT. Generated with:
+#
+#    devctl@4.6.1
+#
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: weekly
+      time: "04:00"
+    open-pull-requests-limit: 10
+    reviewers:
+      - giantswarm/team-atlas
+    ignore:
+      - dependency-name: zricethezav/gitleaks-action
```

Comment:
Just checking, why ignore it?

Reply:
This is from a generated file so I'm not really sure :(